src/contributors/Niharika-Arora.json

```
@@ -0,0 +1,4 @@
+{"name": "Niharika Arora",
```

```suggestion
{
 "name": "Niharika Arora",
```
Add a new line just to make it synchronous with other data files